CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(wxcharts
+        VERSION 1.1
+        LANGUAGES CXX)
+
+################
+# config cmake #
+################
+
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ Standard")
```

Comment:
We want to keep wxCharts compatible with C++11 and above. For this project C++14 and later do not add significant value and I'd like to keep as compatible with the wxWidgets library as possible.

Reply:
Fixed